hokusai/development.yml

```diff
@@ -16,4 +16,4 @@ services:
   metaphysics-memcached:
     image: memcached:1.4.34-alpine
     ports:
-      - 11211:11211
+      - 11212:11211
```

Comment:
Lets remove the port. There is no need to expose it to the host. And without it we don't have to worry about conflicts with `brew` managed services. 

LMK if I'm forgetting something, though.

Reply:
Good call. Done in https://github.com/artsy/metaphysics/pull/5420/commits/0431e7f4166b07921c9a0f4d38a1b6ba493b2dd9